api/src/main/java/org/openmrs/api/db/hibernate/HibernateConditionDAO.java

```diff
@@ -97,12 +97,29 @@ public Condition getConditionByUuid(String uuid) {
 	 */
 	@Override
 	public List<Condition> getActiveConditions(Patient patient) {
+		System.out.println("near query in getActiveConditions in core");
```

Comment:
Its not a good practice to use sysout. Better to use logs instead

Reply:
I completely forgot to remove sysout before pushing. Thank you for pointing it out :) . I updated the PR